src/views/User/User.tsx

```diff
@@ -64,20 +64,20 @@ const inlineBlock = {display: "inline-flex", "alignItems": "center"};
 const marginBottom0 = {marginBottom: "0"};
 
 function getGameResultRichText(game) {
-    let result = getGameResultText(game);
+    let resultText = getGameResultText(game.outcome, game.white_lost, game.black_lost);
```

Comment:
Once I added TS types on `getGameResultText` TS started requiring at least coherent types on the code that calls `getGameResultText`... and here `result` is either a string or an element depending on what happens... 

I changed the code to always return an element. We need to check all places where this code is used to see if it creates any graphical glitches...

Reply:
I'm pretty sure the only place `getGameResultRichText()` is used is the Game History table of the profile page.  Since `PaginatedTable` accepts both `string` and `JSX.Element`, the change you suggested shouldn't be a problem.

A screenshot verifying that the game result column renders as expected:

<img width="817" alt="Screen Shot 2022-01-01 at 5 52 49 PM" src="https://user-images.githubusercontent.com/25233703/147863991-5b607a3f-3278-4d7c-842e-5846232abcef.png">

Reply:
Fun note... I getGameResultRichText was part of my first PR to OGS 😊 good memories, but probably explains the wonky typing!